autoload/intero/repl.vim

```diff
@@ -64,24 +58,46 @@ function! intero#repl#type(generic) abort
 endfunction
 
 function! intero#repl#type_at(generic, l1, c1, l2, c2) abort
```

Comment:
Please add a comment to say what these variables are. (I know you didn't name them, but it's worth fixing up)